core/src/world/place/building/business/inn.rs

```diff
@@ -28,7 +28,7 @@ fn name(rng: &mut impl Rng) -> String {
 
 fn thing(rng: &mut impl Rng) -> &'static str {
     match rng.gen_range(0..5) {
-        0 => animal(rng),
+        0 => any_animal(rng),
```

Comment:
Any particular rationale behind this change? I feel like it scans better as just `animal`.

Reply:
Hmm, no not really. I think I was thinking it might imply to future contributors the existence of other more specific animal generators but I suppose that's a job better suited to documentation. I'm fine with going back to `animal`.